ClientCore/FileHelper.cs

```diff
@@ -83,8 +59,7 @@ public static void CreateHardLinkFromSource(string source, string destination, b
             else if (RuntimeInformation.IsOSPlatform(OSPlatform.Linux) || RuntimeInformation.IsOSPlatform(OSPlatform.OSX))
             {
                 if (link(source, destination) != 0)
-                    throw new IOException(string.Format("Unable to create hard link at {0} with the following error code: {1}"
-                        .L10N("Client:DTAConfig:CreateHardLinkFailed"), destination, Marshal.GetLastWin32Error()));
+                    throw new IOException($"{"Client:DTAConfig:CreateHardLinkFailed".L10N()}: {destination}. Error: {new Win32Exception(Marshal.GetLastWin32Error()).Message}");
```

Comment:
(a) The client heavily uses `string.Format` for localization and your modification breaks it
(b) You can show a friendly message about the error but please also keep showing the error code as an integer